modules/monitoring/run.php

```diff
@@ -6,3 +6,4 @@
 $this->provideHook('ApplicationState');
 $this->provideHook('Health');
 $this->provideHook('X509/Sni');
+$this->provideHook('Themebuilder/LessVarsLoader');
```

Comment:
It's no monitoring hook, but one of the main application. Should be registered during bootstrapping I guess.